wpe/src/main/cpp/browser/pagesettings.h

```diff
@@ -0,0 +1,23 @@
+#pragma once
+
+#include <string>
+
+/**
+ * For now this is just a container to pass settings data around but
+ * eventually this will have 1:1 relation to webkit_web_view_set_settings
+ * and also weak ref to java side object
+ */
+class PageSettings
+{
+public:
+    void setUserAgent(const std::string ua) { m_userAgent = ua; }
```

Comment:
my bad, will change it to const std::string&

Reply:
I will double check the move semantic because I remember there is a new way to pass strings parameters that is quite elegant with the && (but I don't remember the details)